recipes/perl-vcftools-vcf/0.840/run_test.pl

```diff
@@ -0,0 +1 @@
+use Vcf;
```

Comment:
can we drop this file and simply use imports like here https://github.com/bioconda/bioconda-recipes/blob/master/recipes/perl-gd/meta.yaml#L31